array_test.go

```diff
@@ -461,6 +461,128 @@ func TestArray_DeleteFragments(t *testing.T) {
 	require.Equal(t, int8(10), bounds[1])
 }
 
+func TestDeleteFragments(t *testing.T) {
+	// Create an array with domain [1, 10].
+	// Create fragments [1,2] [3,4] [5,6] [7,8] [9,10]
+	// Delete the first 2 fragments and verify the non empty domain is [5, 10]
+
+	// create an array and write 5 fragments
+	array, err := newTestArray(t)
+	require.NoError(t, err)
+
+	testStarted := time.Now()
+	var fragmentCreatedAt []time.Time
+
+	context, err := NewContext(nil)
+	require.NoError(t, err)
+	err = array.Open(TILEDB_WRITE)
+	require.NoError(t, err)
+	for i := 1; i <= 10; i += 2 {
+		time.Sleep(100 * time.Millisecond)
```

Comment:
can you add a note explaining why this is here?

Reply:
local disk is too fast and i needed fragments to have some time distance between them